### PR TITLE
dash/new-packages

### PR DESCRIPTION
**Pull request by karolkolodziej**
Removed `dashboards-plugins` and added plugins to the main `dashboards` package. Extracted `DataGrid` as a separate module and separate namespace.

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

**Review by Vazonik — APPROVED** [review]

File: tools/gulptasks/dashboards/_config.json [hunk 1]
```diff
@@ -11,5 +11,13 @@
     "namespace": "Dashboards",
     "product": "Highcharts Dashboards",
     "readmeFile": "ts/Dashboards/README.md",
-    "typeScriptFolder": "ts/masters-dashboards/"
+    "typeScriptFolder": "ts/masters-dashboards/",
+
+    "buildFolderDataGrid": "build/dist/datagrid/",
+    "bundleSourceFolderDataGrid": "code/datagrid/es-modules/masters-datagrid/",
```
Comment (bre1470) [inline tools/gulptasks/dashboards/_config.json:17]: This should be `code/datagrid/es-modules/masters/` for consistency. Only in the `ts/` folder we have to distinguish more.

**Review by bre1470 — APPROVED** [review]
Setup looks good. 👍 Just one thing to consider.

**Review by bre1470 — CHANGES_REQUESTED** [review]
One thing is missing though. You have to adjust `tools/gulptasks/lint-ts.js` and its calls after excluding data grid with `ts/.eslintingore`.

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Pull request merged**